sdcm/tester.py

```diff
@@ -165,7 +165,8 @@ def __init__(self, methodName='test', name=None, params=None,
         cluster.Setup.set_test_id(self.params.get('test_id'))
         cluster.Setup.reuse_cluster(self.params.get('reuse_cluster', default=False))
         cluster.Setup.keep_cluster(self._failure_post_behavior)
-        cluster.Setup.set_multi_region(self.params.get('region_name').split() > 1)
+
+        cluster.Setup.set_multi_region(len(self.params.get('region_name').split()) > 1)
```

Comment:
region_name might be not set if we use GCE. Then it will fail:

```
2019-04-02 00:46:32,199 output           L0642 DEBUG| Traceback (most recent call last):
2019-04-02 00:46:32,199 output           L0642 DEBUG|   File "/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
2019-04-02 00:46:32,199 output           L0642 DEBUG|     self.run()
2019-04-02 00:46:32,199 output           L0642 DEBUG|   File "/usr/lib64/python2.7/multiprocessing/process.py", line 114, in run
2019-04-02 00:46:32,200 output           L0642 DEBUG|     self._target(*self._args, **self._kwargs)
2019-04-02 00:46:32,200 output           L0642 DEBUG|   File "/usr/lib/python2.7/site-packages/avocado/core/runner.py", line 237, in _run_test
2019-04-02 00:46:32,200 output           L0642 DEBUG|     instance = loader.load_test(test_factory)
2019-04-02 00:46:32,200 output           L0642 DEBUG|   File "/usr/lib/python2.7/site-packages/avocado/core/loader.py", line 263, in load_test
2019-04-02 00:46:32,201 output           L0642 DEBUG|     test_instance = test_class(**test_parameters)
2019-04-02 00:46:32,201 output           L0642 DEBUG|   File "/sct/sdcm/tester.py", line 168, in __init__
2019-04-02 00:46:32,202 output           L0642 DEBUG|     cluster.Setup.set_multi_region(self.params.get('region_name').split() > 1)
2019-04-02 00:46:32,202 output           L0642 DEBUG| AttributeError: 'NoneType' object has no attribute 'split'
```

If the set multi_region is only used for AWS, then we need a default value for region_name here, like this:
`cluster.Setup.set_multi_region(len(self.params.get('region_name', default='').split()) > 1)`

Otherwise, we need to process `gce_datacenter` value.

Reply:
I'll address this one in a separate PR